src/approaches/reinforcement_learning_approach.py

```diff
@@ -0,0 +1,109 @@
+"""A bilevel planning approach that learns NSRTs from an offline dataset, and
+continues learning options through reinforcement learning."""
+
+from typing import Callable, List, Optional, Sequence, Set
+
+import dill as pkl
+from gym.spaces import Box
+
+from predicators.src import utils
+from predicators.src.approaches.base_approach import ApproachFailure, \
+    ApproachTimeout
+from predicators.src.approaches.nsrt_learning_approach import \
+    NSRTLearningApproach
+from predicators.src.nsrt_learning.nsrt_learning_main import \
+    learn_pruned_nsrts_from_data
+from predicators.src.settings import CFG
+from predicators.src.structs import NSRT, Dataset, GroundAtom, \
+    InteractionRequest, InteractionResult, LowLevelTrajectory, \
+    ParameterizedOption, Predicate, State, Task, Type
+
+
+class ReinforcementLearningApproach(NSRTLearningApproach):
+    """A bilevel planning approach that learns NSRTs."""
+
+    def __init__(self, initial_predicates: Set[Predicate],
+                 initial_options: Set[ParameterizedOption], types: Set[Type],
+                 action_space: Box, train_tasks: List[Task]) -> None:
+        super().__init__(initial_predicates, initial_options, types,
+                         action_space, train_tasks)
+        self._nsrts: Set[NSRT] = set()
+        self.online_learning_cycle = 0
+        self.initial_dataset: List[LowLevelTrajectory] = []
```

Comment:
if these are trajectories, and not `Dataset` instances, let's rename them to something like `initial_trajectories`, etc.